stdlib/types.pyi

```diff
@@ -351,27 +348,31 @@ class ModuleType:
     # using `builtins.__import__` or `importlib.import_module` less painful
     def __getattr__(self, name: str) -> Any: ...
 
+_YieldT_co = TypeVar("_YieldT_co", covariant=True)
```

Comment:
Our usual style is to put all the TypeVars at the top of the file, but I actually quite like way you've done it here, as the meaning of the TypeVars is easier to understand if it's closer to the definition. Will wait a little bit to see if any other maintainers have objections, though

Reply:
Speaking generally: I think type vars close to their use can make sense in more specific type vars, especially those with bounds or named `_FlooWidgetT` instead of just `_T`.